crates/matrix-sdk/src/room/joined.rs

```diff
@@ -551,26 +551,37 @@ impl Joined {
 
         #[cfg(feature = "encryption")]
         let (content, event_type) = if self.is_encrypted() {
-            debug!(
-                room_id = self.room_id().as_str(),
-                "Sending encrypted event because the room is encrypted.",
-            );
-
-            if !self.are_members_synced() {
-                self.request_members().await?;
-                // TODO query keys here?
-            }
+            // Reactions are currently famously not encrypted, skip encrypting
+            // them until they are.
+            if event_type == "m.reaction" {
+                debug!(
+                    room_id = self.room_id().as_str(),
```

Comment:
```suggestion
                    room_id = %self.room_id(),
```

Reply:
Updated the other ones as well in https://github.com/matrix-org/matrix-rust-sdk/pull/606/commits/cc6e5b868e4c1007abb2408e6cd8b31560888e01.